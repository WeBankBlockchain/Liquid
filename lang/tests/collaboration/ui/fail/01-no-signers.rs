use liquid_lang as liquid;

#[liquid::collaboration]
mod noop {
    #[liquid(contract)]
    pub struct Noop {
        addr: Address
    }
}

fn main() {}