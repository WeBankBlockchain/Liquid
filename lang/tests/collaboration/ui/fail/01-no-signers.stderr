error: this contract has no signers
 --> $DIR/01-no-signers.rs:5:5
  |
5 | /     #[liquid(contract)]
6 | |     pub struct Noop {
7 | |         addr: Address
8 | |     }
  | |_____^
